alert/alert.css

```diff
@@ -0,0 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap");
+
+* {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
suggestions color can be kept in common variable.css file and you can use them everywhere else